jekyll/_cci2/rename-organizations-and-repositories.adoc

```diff
@@ -10,7 +10,7 @@ contentTags:
 :icons: font
 :experimental:
 
-NOTE: The content on this page is only relevant to GitHub and Bitbucket. Due to the standalone nature of GitLab accounts in CircleCI, no changes are required when orgs or repos are renamed.
+NOTE: The content on this page is only relevant to GitHub OAuth app and Bitbucket integrations. Due to the standalone nature of GitLab and GitHub App accounts in CircleCI, no changes are required when orgs or repos are renamed. To find out if you authorized through the GitHub OAuth app or the CircleCI GitHub App, see the xref:github-apps-integration#[GitHub App integration] page.
```

Comment:
I think we don't have the ability to rename orgs or projects for standalone, so perhaps we mention that in here instead?

Reply:
So my understanding is these instructions for people who rename orgs and repos in their VCS, so I think the text as is is OK? We could add something about how renaming of CCI orgs and projects isn't currently possible too (but will be in future), is that what you mean?

Reply:
@rosieyohannan Ohh, thanks, I was confused, I thought it was referring to CCI orgs and repos. I can't remember if we mention elsewhere about renaming of CCI orgs and projects but I don't think that needs to also go here.

Reply:
OK I've just taken a look at the page with fresh 👀 and it really isn't clear is it! I will try to improve